### PR TITLE
Local storage save

### PR DESCRIPTION
**Pull request by BrianM-85**
Added local storage save for user. Added Create Column functionality.

### CODE REVIEW

File: components/App.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
-import React, { useState } from 'react';
```
Comment (jamescammarano) [inline components/App.js:1]: you import usestate twice i think

File: components/App.js [hunk 2]
```diff
@@ -88,17 +95,47 @@ const App = () => {
   const columnComponents = getData.columnOrder.map(columnId => {
     const column = getData.columns[columnId];
     const tasks = column.taskIds.map((taskId) => getData.tasks[taskId]);
-
+    debugger
```
Comment (jamescammarano) [inline components/App.js:98]: u no

File: dist/main.1f19ae8e.js [hunk 3]
```diff
@@ -40930,20 +40940,43 @@ var App = function App() {
     var tasks = column.taskIds.map(function (taskId) {
       return getData.tasks[taskId];
     });
+    debugger;
```
Comment (jamescammarano) [inline dist/main.1f19ae8e.js:40943]: oops

**Review by jamescammarano — CHANGES_REQUESTED** [review]

File: components/App.js (hunk 1 above)
Comment (BrianM-85) [reply components/App.js:1]: Talked through the confusion here

File: components/App.js (hunk 2 above)
Comment (BrianM-85) [reply components/App.js:98]: doh

File: dist/main.1f19ae8e.js (hunk 3 above)
Comment (BrianM-85) [reply dist/main.1f19ae8e.js:40943]: doh

**Pull request closed without merging**